### ISSUE TITLE
java.lang.RuntimeException: Unmatched delimiter: )

### ISSUE THREAD

**Issue by kenfehling** [boot-clj/boot#349]
I just updated to version 2.4.2 and I'm getting this error when trying to use boot:
```
java.lang.RuntimeException: Unmatched delimiter: )
                      ...                   
 clojure.core/read-string     core.clj: 3663
boot.util/read-string-all     util.clj:  205
          boot.main/-main     main.clj:  148
                      ...                   
         boot.App.runBoot     App.java:  361
            boot.App.main     App.java:  441
                      ...                   
         boot.Loader.main  Loader.java:  253
```

**Comment by martinklepsch** [comment]
@kenfehling what are you running exactly? are you sure this works with `2.3.0` and doesn't with `2.4.2`?

**Comment by kenfehling** [comment]
I'm trying to use this: [Creating Packages](https://github.com/cljsjs/packages/wiki/Creating-Packages).

I'm running these commands:
```
boot -d cljsjs/boot-cljsjs -h
boot package build-jar
boot --help
```

All of these give me the same error.

I know it used to work, but I'm not sure with which version. When I try with 2.3.0 it just tells me to upgrade to the new version.

**Comment by martinklepsch** [comment]
What's the result of running `boot -V`? How are you downgrading to 2.3.0?
(hint: use the BOOT_VERSION env var.)

**Comment by kenfehling** [comment]
The output of boot -V is:
```
#Wed Nov 25 17:19:25 EST 2015
BOOT_CLOJURE_NAME=org.clojure/clojure
BOOT_VERSION=2.4.2
BOOT_CLOJURE_VERSION=1.7.0
#App version: 2.4.2
```
I was downgrading to 2.3.0 by downloading [boot.sh](https://github.com/boot-clj/boot/releases/download/2.3.0/boot.sh) and then running
```
mv boot.sh boot && chmod a+x boot && sudo mv boot /usr/local/bin
```

**Comment by martinklepsch** [comment]
@kenfehling you can set the env var `BOOT_VERSION` to `2.3.0` but use the same binary that you used with `2.4.2`. I'm using `2.4.2` myself and everything is working well for me so I think it's most likely that this is caused by your own code.

**Comment by kenfehling** [comment]
You're right! The problem was in my ```build.boot``` file.

I was confused by the behavior because even running something like ```boot --help``` was affected by my file's syntax error. When I ran it in an empty directory though it ran successfully. Also the stack trace didn't mention my ```build.boot``` file even when I ran ```boot -v``` to get the verbose stack trace, so that threw me off as well.

In the end I figured it out though, thanks for your help.
